FruitFruit/FruitFruit/Controllers/OrderViewController.swift

```diff
@@ -81,14 +72,44 @@ class OrderViewController: UIViewController, UIGestureRecognizerDelegate {
         guard let blackColor = UIColor(named: Constants.FruitfruitColors.black1) else { return }
         
         fruitOriginLabel.text = "\(fruitSaleInfo.fruitOrigin)에서 태어난"
-        fruitOriginLabel.textColor = blackColor
+        fruitOriginLabel.textColor = UIColor(named: Constants.FruitfruitColors.black1)
+        fruitOriginLabel.font = UIFont.preferredFont(for: .subheadline, weight: .bold)
+        fruitOriginLabel.translatesAutoresizingMaskIntoConstraints = false
+        fruitOriginLabel.topAnchor.constraint(equalTo: view.topAnchor, constant: 395).isActive = true
+        
+
         fruitNameLabel.text = fruitSaleInfo.fruitName
         fruitNameLabel.font = UIFont.preferredFont(for: .title1, weight: .bold)
+        fruitNameLabel.textColor = UIColor(named: fruitSaleInfo.fruitType.fruitColorName)!
+
```

Comment:
```suggestion
guard let fruitColor = UIColor(named: fruitSaleInfo.fruitType.fruitColorName) else { return }
// 옵셔널 바인딩 -> 이후 fruitColor는 UIColor 옵셔널이 아니라 UIColor 변수입니다
fruitNameLabel.textColor = fruitColor
```

로 주실 수 있습니다. guard let 이외에도 디폴트 값을 줄 수도 있습니다.

```swift
let fruitColor = UIColr(named: fruitSaleInfo.fruitType.fruitColorName) ?? UIColor.black
```

으로 주시면 앞의 UIColor가 nil이 될 경우 뒤의 값이 디폴트로 들어갑니다